app/controllers/owned_tag_sets_controller.rb

```diff
@@ -201,18 +201,35 @@ def do_batch_load
       redirect_to :action => :batch_load and return
     end
   end
-  
-  
-  
-  
+
+
+
+
   protected
-  
+
   # for manual associations
   def get_parent_child_tags
     @tags_in_set = Tag.joins(:set_taggings).where("set_taggings.tag_set_id = ?", @tag_set.tag_set_id).order("tags.name ASC")
     @parent_tags_in_set = @tags_in_set.where(:type => 'Fandom').value_of :name, :id
-    @child_tags_in_set = @tags_in_set.where("type IN ('Relationship', 'Character')").value_of :name, :id 
+    @child_tags_in_set = @tags_in_set.where("type IN ('Relationship', 'Character')").value_of :name, :id
+  end
+
+  private
+
+  def owned_tag_set_params
```

Comment:
It might also be necessary to include the parameters for the fields in [_tag_set_association_fields](https://github.com/otwcode/otwarchive/blob/master/app/views/owned_tag_sets/_tag_set_association_fields.html.erb) and [_tag_set_associations_remove](https://github.com/otwcode/otwarchive/blob/master/app/views/owned_tag_sets/_tag_set_associations_remove.html.erb) -- we didn't have a test for that, and it failed when I tried to testing manually. /o\ 

I should have a test for it in #2788.